apps/omar-ui-app/grails-app/assets/javascripts/omar/services/map.service.js

```diff
@@ -176,6 +177,89 @@ function mapService(stateService, wfsService, $timeout) {
       name: 'Image Footprints'
     });
 
+    var footprintsSource = footPrints.getSource();
+
+    /**
+       * Renders a progress icon.
+       * @param {Element} el The target element.
+       * @constructor
+       */
+      function Progress(el) {
+        this.el = el;
+        this.loading = 0;
+        this.loaded = 0;
+      }
+
```

Comment:
Do the two functions below increment for every tile loading/loaded? i.e. what is the max that the loading or loaded counts will ever reach? Or is it just 1 or 0?

Reply:
Yes, the tiles increment for all the tiles. I am not sure what the max would be.  I guess that would be determined by the size of the tile(s) being loaded, and the viewport where they are being rendered.